include/sys/zio_impl.h

```diff
@@ -24,7 +24,7 @@
  */
 
 /*
- * Copyright (c) 2013 by Delphix. All rights reserved.
+ * Copyright (c) 2012, 2015 by Delphix. All rights reserved.
```

Comment:
good observation.  This change was directly from Delpix in their commit to Open ZFS.  https://github.com/illumos/illumos-gate/commit/4756c3d